### PR TITLE
Create howl.ms

### PR DESCRIPTION
**Pull request by Wyndfall**
Howl new character (From howl's moving castle)

### CODE REVIEW

File: CommandHelper/LocalPackages/commands/buffs/howl.ms [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+                "/dismountplayer",
+                "/disguise"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 20 true",
+                "/effect ". @plr ." 10 9999 1 true",
+                "/effect ". @plr ." 11 9999 5 true",
+                "/effect ". @plr ." 12 9999 1 true",
+                "/effect ". @plr ." 13 9999 1 true",
+                "/effect ". @plr ." 21 9999 19 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/howl.ms]: remove the comma on line 29

File: CommandHelper/LocalPackages/commands/buffs/howl.ms [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            'powers': array(
+                "/buff",
+                "/jump 0-5",
+                "/run 0-3",
+                "/haste 0-5",
+                "/fly",
+                "/flyspeed <number>",
+                "/carryplayer <player>",
+                "/dismountplayer",
+                "/disguise"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/howl.ms]: requires proof

Comment (Wyndfall) [inline CommandHelper/LocalPackages/commands/buffs/howl.ms]: Proof for carry player, jump, run and fly:http://video.disney.com/watch/that-s-my-girl-4d91a69e7a08fad88e31f9ed
Proof for flyspeed:https://www.youtube.com/watch?v=UibodUGoL4M at 1:24
Proof for disguise: http://www.wallpapersxl.com/wallpapers/1680x945/howl/181248/howl-181248.jpg (This is the disguise howl uses to watch over Sophie while she visits solomon for Howl to tell her that he will not come.

**Comment by Wyndfall** [comment]
Also he holds a fire demon in his bear hands... Pics: https://s-media-cache-ak0.pinimg.com/736x/5e/7a/96/5e7a96b1030035ebbe1c7681cb79480b.jpg , data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wCEAAkGBxITEhUQEhIVFhUVFxUVFRUVFRUVFRUVFRUWFxUVFRUYHSggGBolGxUVITEhJSkrLi4uFx8zODMtNygtLisBCgoKDg0OGxAQGy0mHyUtLi0tLS0tLTAtLS0tLS0tLS0tLS0tLS0tLS0tLi0tLS0tLS0tLS0tLS0tLS0tLS0tLf/AABEIAKgBLAMBIgACEQEDEQH/xAAcAAAABwEBAAAAAAAAAAAAAAAAAQIDBAUGBwj/xABKEAABAwEEBQcFDQYGAwEAAAABAAIDEQQFEiExQVFhkQYTIlJxgaEHFTKxwSMzQlRicoKSorLC0dIUFhdTk+EkQ2Oj8PFzg7NE/8QAGgEAAgMBAQAAAAAAAAAAAAAAAAMBAgQFBv/EACkRAAICAQQBBAEEAwAAAAAAAAABAgMRBBIhMVETIjJBMwVhcZEUQuH/2gAMAwEAAhEDEQA/AONGBuxHzDdicKCckimRAhbs9aHMN2etLBRkqyiiMjfMN2etH+zt2etLBRqdiI3DYs7dnrRfs7dnrTqBUbEG4b/Zm7PEohZ27PWnwiKrtLZGXWdtNHrQlszQAaetOIyclGwMjEcDTqSXWamkKTCKFHKKqHEnJFEI2J0WZuzxKVgTirgka/Zm7PEpxtmYCDhBoQaGtDQ6DQ1oUoJxVZJcX067JIG8xZZIJwRUYy+NzdDhic6u8Zald8l7uue0wczMxsFqDcIe6SRrZHUykbV2Gust4ZLFFEluvKwmy6nh5wdQ5PclrqtMghNna2eIv5+ITSOBYG9GRjsWbCXR6NtCtR/DK6viv+7N+tZjyH2PpWmemVI4hxc934F1ZIm3F4THRSazgyH8Mrq+K/7s360P4ZXV8V/3Zv1rXoKm+XkttXgxdr8md1hji2zUIFR7rLq+ks7NyFsNW4bPpdSmOTOoNPhbl1YiuSyZ6LmnqvbXsDwHeFUb5eQ2rwZaXkHYWmjrPQ/Pk/Um/wByrBWnMaq++SfqXRbfYxIAK0IOR3aws3KwtkwnTQjgR+aUrZP7GbI+DP8A7kWD+R9uT9SH7kWD+R9uT9S0KNT6kvIbI+DO/uRYP5H25P1IfuRYP5H25P1LQoKd8vIbI+DOs5EWDGBzGRa4+nJpBZT4W8pM3Iy78YYINAJcccnc30tOdeG1Wl72p0eBzNJLm6KnMVo0bejvTlka4BgcauGMOO11RU71bfLHZTZHPRVfuRYP5H25P1IfuRYP5H25P1LRIKvqS8l9kfBnf3IsH8j7cn6lKsnJCxNBDYaCtfTk2D5St2moqE9DoSb5y2dmXWRj6XRwNBEjXfRlYQSqI2hLAToxyUbwJDUeFS4LOXEACpOoKz8wy0rhHZUVWuOmbRnlckUNESmWqylpo4EHeopak2VOJeNmQgielIOCz7Ru4Ql4ckA1OYVKgRuGJAUbQnS1JLVEoNFlIQgUaBSWhqYYS0gJSWywEAgCjKgDuXklsPN3cx5FDM+ST6OIsYfqsB71s1zHkL5RLOyCOy2r3IxNDGyBpMbmtybipUsdSlToOnLQug2S9IpRWFwlG2N0bvDFXwWKyL3Ns0wawTUEiKUOrSuWkEEEcUtULgWZvmKj5BtBI+kPzWmVVf0FQ14+afZ7UASYn4mh20A8RVUl92aj2v6xI8Bl4K0u11Yoz8lvgE3eMWJ0I/1M+wRyO/Cs64Y36G5LraWNaMnClXba6aqNeF24aGME6iNJrtVzXOmvI8a09R4I0ZZJk3CmRyppRKw5QWenTAyINfnf9epV7UxPKIItrDucgwmnuvSyrVvNSVGjLVnuSpMpafLJHY6In7zXJ68bHIWxuiBLmyBwA10ZJUdlEq8oQJGPGhwaR2dMD/6pmPaZ/Wi7NoEaJGlmgas/o959ZUmLQo8Gjvd94qTDoSr/AImTWfiOBJTQioltXokY2BSIGJkKbYBVwG0j1rbpkmzPa8I1dyWAMYHEdJwB7BqCtiG4RQHFnXRSmVKeKQ1OOdloGXj2rsxWEc+TM7ynAo0a8+2izD2q2vGUucSdarnNWHVIfSyPhSgxOYU/Y7Pie1u00WOFe6WB7nhDMUBKsbNdEjtVBtOS0llsjGCjR36z3oWq0hmqvgulDTxRmla2Uh5Pu6w8VW2+7nx+kMtRGYWi86jW09xUK9Lya9hYGnPWaZU2KLaoOJMJyTM0WpKkOYm3MXGnDk3xsGiUpqGBABIlAapimpTmIom1KfelNF0xhHG4tIc0lrhoc0lrh2EZhKLEkKCTrXkm5UTzuksloeZMDA+N7s30Bwua53wtLSCc9NarpK435GY62yR3VhP2ntHsXZFjtSUjTW/aBN2iLE0t2jx1eKcQSy5UXN7yAdTpG/VleB4AJ+0DNh2PHi1zfxJyxspz7dklfrNa/wDGU3atA+fH4yNHtSJL3DIvgDffHfMZ96RPJke+H5jfvO/NPKpZEG+46wP3CvDT4VVWyyHAx4BJJdXsA/sVcXqfcZPmO9SesseFjW7AAeGftUp4RDE3ZRoa7TQ13+g7JUN6uBgheBQCRrKbBzmEDiAFbWy1c2+jW16TS1uQFS06TqGar7zPuWAZmrSctLucDiaas6rXFpwOLGqa1GfogI05aYCxxadIp4ppIO0N2fR3u+8VKh0KJE7IDa5w+8fYpUWhKv8AiZNZ+I4M5GEKIFehiZGCqkQPooqcaVqqntEzjk19jv4UAe012ime+hVrBamvFWmu3aO5YKN6srBaA0OcXODhTBQVDjUVDjqFK8F1arlLgwzrwPW+zOaaEd+o9igOYtbY7WJG4hp1jYUi22FrxoAdqPsKvOtTKxltMrgT1mq1wcNIzUo2Ug0IU67oXMe17QCQajEAR3g6UiFLjIvKaaJ9itgdpFD4KVabHjG9O3ZdBydka50FajPXlTgtHBdRI0LVOaj2ZHYs8GDmux2xQpbtf1TwXSnXRuRC5NyS5Vvsur2jlrruf1TwTElgcNLTwXW/MCYmuMhL2UsstQzkbrMUy6JdNtVyg6WgqivDk/rbwP5qk9JGS9rHw1HkyEbKInHNWFosjmGhFFAkbmuZbQ4vk3QtTQglEUdEWArM44HKROuS95rLM2eB2F7cjUVa5p0scNYNO3Yum2LyuwUHPWaVp+EYyx7e4OLSuTAJTcsxQnVXRXVXckzrUuxkZtdHpO6LzjtMTZ4sWB1aYhhORoajtUxQbjdCYI3WcARObiYBXIOzI7akqaCsT7NY1Zh0px80/wC20fhUe2xnB9KPwexylWP32QbWs/EEd6to0U6zfvN/JJkuSyYzZme7CvU9oI9aXNHQ07OJS2j/ABFNjD62qRaW+jvcPUVXHAbuSptzasLTrLBxe0e1PpFvoS0bXs+ycf4UtVLooL5f7s3tb91xTT3esesJV7ub+0tDntbU5FxoCeadQV1adaK0xltQ4UOw9qfDoXJcjl9e+nsChK0vOGrpHdVrOJP/AGqmV+EFx+CCeAqqLoYMRfA+k7jX9Smw6FEhZQtb1WAez8JShEXE56DTwB9qpf8AEx6zio4miISqIw1dxMztDLgjCUUuSFzQCRTEMTd4qRUcDwTosW0E0qUbY8sbEXdBpJa3YXaT4BM2WcNJJa11WlvSrlUUxCh0jSEhpT4TaXDEyiTbPOWmoJHZktDYbxGEYnEuJIIIFA2gwkHWa4stwWds0QLXOxNGGhoa1dUgUblq0qbYW1IXT09kvsyWRRr4YGuINAe0V4qzsF3N2KHdbSANWVD2bCtBY8losnhcGOabLa7bvGVd391roYhTLRuWXsNporuzWtcfUqUmN07jDsnPsjDpaOHtTRu+PZ4p2O1A6+KElp2Cp8FkzNcGzFbWRv8AYGb0xPdYIyz8EpznnXTw9SbII+EOKunLyUcIv/UorfdLhnhPDJVcl3YtS2sUhOvxqlGNnwmNO+ma0R1Uo9ivRX0zlF83GCMx37FiLwu1zDQjsK9DWu6YZAcuBIPBY6+uRxNcBDhsOR7joPgtUdRXats+GEVOt/scZfDRMPaV0a2XFgGB8dO0eIOvuWMvaw826mrUqXaRbd0Waa9Rl4ZVByNulAhBoXJnHBujLJs/JfifbOYMkgjMMr8DZHtZiaY6Owg0rmdWtdbumLAXxk1zDm10lpqOIIIPdtXJ/JIz/Hk7LPN4uiC6ja7SI5onE0b02uOxrsOZ3AhpXNveLMG2pZgWtk9+f8xnrche56LfnM++1Js591k/8bT4uTV6PrRv+oz77CfulIk+RiXI/C6toefkN8S78k9apaYaajVQrKelIfkMH2n/AJhGTVUb4LKPIzL6TPnE/YcPanU0/wBNu5rj4tCdVS5g+WUlZwPnepv6SmruveRoELhzjCQ1uInFHUgAsds+ScuxQuUdoxWwNroa93fl7JEVhFZIx8tg4vC1xWIoS+zpksQcCDrpXuWVvXS8bXYeLqLWrHW6QY89Ac55+awH2lqzQ7GsKM5vcdvg0fni4pyyPIb6LidJwtJALs6V3VTIicWtjaOm8gU1AnpPJ2ADF4LSwQCNoYNQzOsnW49pSdVYoRwZdUs1nnEFKxZJCestnc9wYwVcdA20FT4BegXJmbwM0SnIjlmgHA6CFblFeGLeG4RSuLPEDo1Up4pLU+2wTEVEMpG0RvI40QtErgxsb2BmEk1LS1xrqcTpA1J0JJi5RZJuy0Bji4ivRcBo0uaRUgg7Vprls4DA6mbs6rHROWkum9GhoY7VoO5dfSTWMGC6P2bGxBWsKz932oOFQajR2E/9FXdnkqn2JmQnMkopUVsIUFpQlfQb9SzSS+yyjnotY70zwjM6FcWO8GeiCCdZ1dyxULi2tNYpXZtonYZy01BWacIyNUa9q47N3aKkZFVrgUxdt8A0a5WzoQ7Nqz4cOGRJNkOxvIVg2bVpTEVmAPSUW220tOFopvQ1vfAL2LksZWgZ1z8VXWu1vbnQOGw5HiEVmq4VT7odqlJRfJVyzyiubbopfc3Cjj8B9KHsOgrJcqeSLZRWJ2Bw0B1S3jpHirDlHCAVIui3GWI4jVzDhJ2igIPb+S1wTisxfBClnk41eV2yQP5uVha7SNYcNrToIUV2iq7ja7pitMRimbiGo6HNPWY7UVyrlTyXlsb+l0oj6Euo/Jd1Xbterdy7ZpyaOtCDUcl75HY62md/Vgp9eRv6Vu79PTaNjfWf7LF+SDJ8563Ns8JX/hWvvd1ZTuAHh/dcjUc2nQpXsJ/JfRLpyaAM8gKaBsGal2g1dH84k9zH+0hReTJ6Ex+VT7LPzUl/vjRsa48S0JEuxiHIX5v+iOAB9qWmbKa4jte/7JwfhTwVWWRHidWR56uFn0qY3eD2cE5aX4WOOwH1ZKPdLsUfOfzHOkHY49H7OFRuU9r5qzPdrplvOod5oO9TjnBH0cytEhdbHPocIY8V1VLmYR24WAq4uJuK0QDa9p+oDIfBiiNipE0/LPChYPU1XHIqHFaGu6kbnDtcAweDnLS5e1/twK28/wAm6mfRpdsBPALFSmr6dgO4DpvPZ6A+kFq73lwxO35fn4LOXDZOee57h0GnpfKcelg7BVoPzANazqSjFyY1lzc1loOdcM3ej8lmnidPDYmbRelXEMFQMq0Jrw7U9e9qOULM3v8ABusn/nrCKCEMaGjVr1k6yVk9J2pzkZdfPbVhHntEjR0XozOzqnk9Fhks8RmbHNO2rA1zI3GOjnlrA06MjXEdNdOQW4numJ9C6xWY0oWkloc0g5EERZHsK4BdF4yWaUTRUxAFpDhUOaaVaeAz3LoN0eUmPJsmOI7ffI/zHDvSLlOT3djanCKx0dDwvGgSjdjbIOyrjiUaS2MNQ4NeBkaAucDsfHhqPFQ4uUDbRE7m3MdVpGJjtBI1gZj1pVmvwtAa9lKZChA4DQe5VhPCw0XlHPOSNPcF3T5mzw4jsYIpO8DC4d6qbT5PLKTVjpWbg4EfaBWubbYpBSo7HDwzROstPQJbu0tPcdHdRPqvlHpirKk+0Y2DkjzJOGYuBFAHjDQ1GZLdOvKmtWVnuuZuoO+aa+Bora0tk2NPYSDwIp4pLLcB6VW7ajId4y8V2qdVOcMNnJ1FCi8pEI4h6TXVAAzB0AUA4KNPJV3YPWr20Xi0xuwuaaUIo4aQQRr2hZa9LxBmc4/CoT2gUNN2SbmUvoTWoxZOZmjfFnU1pu0qJZ7Y061OFpZTNw4pbTRpyHZHAaju6X9lorlvU+gRQblkLRa2NNcQ4q2snKizBoafS1AaT2DWqyrcl0Lsnjo1E1raDXUdai2sMd0gQqZ14SyikcRA60nRHDT4JuG5Huzklca/Bb0G8dPiEl+nX8pYKRqtt+KLJ18RRZFwJ2DM8FHlvqZ/vcRA2u6Pgc1Kst1MZ6LQN+vjpUptlWeWsqT4Wf5NUP06bXuf9GQvC67TMaukY3dRzvyUi57slgYWdF5c7EXElmoAACh2eK0/7MgIFWX6jJrCNMdBGJCs2IAAsPcQ4ew+Cl2iyMlYY5GNexwo5rhUEbwnmxpwBc6djk8m2MMIx/Jzk02w2iSNji6OSkkdc3NDWuY5hOumMZ6w7tSrxk6cjthce5v/AEtTbIqtxAdJhxDf1m97ajtodSxF6zgty/zXsYN4keMR+qSs8sylkdHhYNFyZbSAfKc6va04PWxSWSe7SHUyOMd5MjnDgGcU3c1GwMOgGrv6j3O9bkxZXVbO7Tzk5YOxvNw+GFx4pL7ZcsbK2jGg9UV7SKk8aqFe1qIs5IyfJhjZTU6U4Qe4EnuKmWs9EgZF1GjcXZV7hU9yqrW4SWyGAejA10zhqBpgjHaA/gSoSyDLmNgaA0aAAB2AUCxnlGt4a1keoAyO7BoHaaGnYtdI8Y8zQMaXOOoYqgV7muXGOWd9ftEpc09FxJHzWnDGODSfpJtEN0ilktsTUw8zaIf8LIHlrWkxHKUFtPgnOhI8Va8gG5PftOAVBGTAHE9hxs4LkcDy1wcCQRoIJBHYRoXZ+RMrv2bHK4lwFXOdpOQe45bMYZ/61adLqi+cp/3/ANKQt3vlckvlGHP5uBgqXnPYG/CcdwGLiFIcWWaGgAAbWg0Vcakk9pqSe1Pwx1JkcKOcAKdVozDe3Wd+2gVQ8meYk+9xnRtdpA9p+jvWHm6agukPbUVljl3wnOV/pvzz0gahTbrPcNSlFKSSt1sVGvCOXrHmDZ53SmogEsBdIkCQllEgBUMjmOEjHFrhoc00I7xq3LV3Vy8lb0bQwSN6zei/vHou8FlAkkKHFS7JUmujrF1XlZ5x/h5cLtcZFCO2M6t7T3q0it0keRqBtFXM79be/LeuJNdQg55bDQ9xGhai6eWc8VGyDnWaOkaSAbn6+/iluproYrPJ1OC9w4ZgHe0+xSGTxO0OodhyWMs3KSxTULpGxu/1Pc3A7A/RwKt2MJFWPDhsJDgex4zHbmojZKBLipF1JdcTsyxpO0taTxoo0/J+F4oYx3VHqUKOZ7fgvHzDiHAZngpMN6O0CQV2OFHcDQpy1k19i/8AGi/oiyckIvgOkb2OBH2gUuzckYx6bpHne7CODaK1ivJ2trTxCmQ3nHraRwKl/qNmOwWkgVsXJazj/KB7ST6yrKw3LHH6EbWV04QB4pTr5aNDCe0gfmkG+9kf2v7LNPVWy7Y2NEI9IsY7OAnQ1Uj74edAaOJUaS3SO0vPdl6khtvsckjRveBpIHaaKNJeUQ+FXsBKzpNdPigoAu33wzU1x4BNG+vkfa/sqlBAFzFe7T6TSN+lWbTXMLJK6uS01HNnVmOzWEAWiw3K27nMngka081jlkcRoa8Ruo0jYcTnDeDuW5UK+mNdC5rwCHFrc9WJ7Ri7q17kAQZpBDAwu0Ma2vYxuN3gwpm4RWGDewzGuRrJWhI343LLeUDlQzmn2eOvOPjIAOQbzji2pOohjXCmrnO1E7l3Z4qhjXP5vE2MAUa4RgRxDF1SOcfX5QS1VJx6JdkU+zYXjb44g+aQ0ZA0udtLyPRA1mhAprMg2LL8hrwqy1XjaDTnHOdtDY4qZAazV2EUzOHcueXpfM8/vjzSriWioaS5xcSRr1DcGtGpNC8peabBzjhG1weGg4ekCSDlpo4lw3knStEdNiOBTu5ydA5fX9gh/ZGmks1Hz0NebYaDmyRrOENO5p2hczkbQp/nK6c/FMyjWn11qCwJnLc8iWhWUF+WhpaeccQ1zXYK0Y7A8yAOA09Jzj3qvaEKK7SfZCbR3e475itUQliPzmn0mO1tcPboOpM2uyuicZYxVpzez8TT/wA/LjV3XhLZ3iWF5Y4bNDh1XDQ4biuncmOXMVopHNSKXRmfc3nVhcdBPVPcSuPbpbKJb6+UbIWxmtsi5s1qa8dE56wcnDtHt0J0qvvixYHCRmVdBGRaddNx2IoLe6nSaCRrBpXtG1TLURnXlmTW1tV8HCwlBEEYXZKAKJGUdEAJCBQojQAghLacglHQgEZASQnLNK+M1jc5m9ri31IqIFQSX1h5YWyPJzxINkjRX6zaHjVX1l5fRuymgcN7C14+q6h4VWDRqjriyym0dVsN82OWnNTNa7UKmM/Ud0TwKuWA0zNe6i4gW1UmyXlNFlHK9o2Bxpw0Jbp8MurfKO0ILkQ5RWr4xJxRt5R2sf8A6JOIPrCr6DLeqjriC5N+89s+MP4M/Sj/AHmtfxh/2fyR6Eg9VHWEFyR/KC1nTaJO409Sbde1oOmeX67vzR6D8h6qOv0QXG32yU6ZZD2yP/NS7oviSGQPD3UqMYPSxNrmKHch0PyHqo6ylwylrg4aQmo5A4BzTUOAII0EEVB4FKSRpqoZA4Bw0FG9gOTgDmDmK5jQVTXNa6HmzoOjcdnertQBjfKTyaNpibNEBzsWkHLHEfSFdAINHVPytq5Pe90WiyvEdohfG4ircWhwrSrXCod3HKoXopV178m47dYRA80Iq6CTSYyCcH0cNGkbD2LRTJ9CLV9nntGAnbfY5IZHwytwyRuLXt2EbNoORB1ggpoFaBQRCFdaNySUALcQRv3IgElidcKGiAEUR01JVElAGiunlTNG0RSudJEDkDm5nYTm4biezYtrd1obIzGwhzToI7BwO5crCtrnJDHUJHS1Ej4LVz9ZpouOVwU1FzVWGZdGjDUeFdEsEAjQSiEEiSEVEoIKACKFEZCSEALCIowiKADQCCCAAiojCIoAMNSgxBpSggAqIqJSJABoIURhAADUYCUFaXBdJnkzqI25vO3YwHade4IA1nIW8C6EQuyLK4DqcyursrT/AIaahYrCY3dHIsPRpoFNGWyngVr7HaRIxsg0HVsIyI7iCs10NryOqnuWB4LQ3Za+cbn6Q079hWeSoWuLg1gJcdAGk7c9Q2kpI40NpmLnCCM+6PyqNMbfhSHZQaN5G1XzGAANaKAAADYAKAKvuW6hC0kkOkd6bho3NbX4I8dO4WS0QjtRmnLcyivnk9ZJZRaZrPHIQ0MeXtxUYMw6nyamu47lXXt5OLunGUXMu1Pgdhp9E1ae8LXKHYYyHSDEcDSGsb1ata8iuv0qDYBRMKHHb98lVsiq6zubaG9UERy/VccJP0h2LD2uyyRuMcrHMeNLXtLXDuK9TKm5UcmrPbojHM3pAHm5QBzkZOtp2aKtORopyB5uASwFY3/cctjndZ5hmM2vAo2Rp0PZu3aiCFAorAAJFEsIkAGArW6R0D84+pqqQre6D0D872NSNT8DNq/xmJF5nqjih5zPVHFEgnmkHnI9UcUfnQ9UcUEEADzmeqOKHnM9UcUSCAB5yPVHFDzkeqOKCCAB5zPVHFDzmeqOKCCAB5zPVHFH5zPVHFBBAA85nqjii85HqjigggAxeh6o4o/Op6o4oIIAPzqeqOKHnY9UcUEEADzseoOKHnY9QcUEEAA3seqOK0Nj5emJgYyzMAHyzxOWlGggBm0cuHucXcw0V+UfyUm7vKJJFiAgaQSDQvIoaUJ0dnBEgoaysMFxyiafKlJqszPru/JWF0+WR8DaNsMbnn03mV2J1Po5DYBkggoUIrpEuTfZP/jzN8Rj/qu/Sh/Hmb4jH/Vd+lBBTggH8eZviMf9V36UzF5cpQ5z/wBijq4j/NfQAACgFNwz/JGgjADv8eZviMf9V36UP48zfEY/6rv0oIIwBn+WPlONvjY19jYx8bqtkbI4kA5OaRTMHLvAWT89nqDiUEFIB+fD1BxKLz2eoOJQQQAPPZ6g4q5ua9yWE4B6R1nY1BBI1HwM2r/Gf//Z Wow thats a long link ;-;. He has the resistance also, because he is resistant to Solomon's magic when she catches him is disguise: http://vignette2.wikia.nocookie.net/howlscastle/images/a/a2/Madam_Suliman's_spell.jpg/revision/latest?cb=20120509231025 (This spell is suppose to weaken wizards, but howl is resistant to it.

**Pull request merged**